crates/libs/bindgen/src/extensions/sockaddr_inet.rs

```diff
@@ -0,0 +1,42 @@
+use super::*;
+
+pub fn gen() -> TokenStream {
+    quote! {
+        impl ::core::convert::From<::std::net::SocketAddrV4> for SOCKADDR_INET {
+            fn from(addr: ::std::net::SocketAddrV4) -> Self {
+                SOCKADDR_INET {                    Ipv4: addr.into()                }
```

Comment:
You might want to run `cargo fmt` here or is there any particular reason for the number of spaces here?

Reply:
I ran `cargo fmt`, committed and pushed!  Fixed it.